openshift-origin-rhel/azuredeploy.parameters.json

```diff
@@ -0,0 +1,57 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "masterVmSize": {
+      "value": "Standard_DS3_v2"
+    },
+    "nodeVmSize": {
+      "value": "Standard_DS3_v2"
+    },
+    "openshiftMasterHostname": {
+      "value": "demomaster"
+    },
+    "openshiftMasterPublicIpDnsLabelPrefix": {
+      "value": "demomasterdns2016"
+    },
+	"nodeLbPublicIpDnsLabelPrefix": {
+      "value": "demonodedns2016"
+    },
+    "nodePrefix": {
+      "value": "demonode"
+    },
+    "nodeInstanceCount": {
+      "value": 2
+    },
+    "adminUsername": {
+      "value": "hwadmin"
+    },
+    "sshPublicKey": {
+      "value": "ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEAu4dL6R9cvph2yRzxP5jh3zE78Ca9CTjyRv3YvKRrOP06zQghBhna65TE85OGkLmV8sJRtH3ug3xlpX//SGBTN6ofO9OzcnikF8Yo8pK64kA/y+v1Mkrg0ZZfl2Td5TXgcw3G9/SOsom8E+syPnZQvNsQhROgF43aSC52QlpIgzeqHW41RRn0iqoXruhxZtOZN3phbR7HCOpdtzavm0PyJzvP/0fPktpdtrUQkWoumcpa8MBvxuEfLS/LbdonNY4i21CKNnnHVdVYEOZtc4lMixm7SKXFL3Ou6aWsBaJpxFbL+8rr17bZ1y6agQaJpbq81zAghRUC8rnNubw27KrCUQ== rsa-key-20160531"
+    },
+    "sshPrivateKey": {
```

Comment:
This is likely a security concern. Instead you should ask the user to upload the private key to a Key Vault and then reference it in parameters file. Here's an example for a password but can you try it for a SSH key? https://github.com/Azure/azure-quickstart-templates/blob/master/101-vm-secure-password/azuredeploy.parameters.json#L8